text/0000-security-team.md

```diff
@@ -0,0 +1,181 @@
+* _Once the vulnerability is mitigated the core team may optionally share some of the details with the community_
+
+### Proposed Process
+
+* Grin's security contacts, all members of the security team, make initial assessment of the validity of the vulnerability disclosure
+* It is determined that the disclosed vulnerability is a critical inflation bug
+* _A conversation and plan of action is quickly and unanimously determined and executed on by the security team, following Grin's responsible disclosure policy_
+* _Once the vulnerability is mitigated, the security team must disclose all information that is not still security-critical with the community_
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+* The core team is already busy
+* Even though it decentralizes the core team, it potentially creates another point of centralization
+* Adds resource overhead to the project that _must_ be upheld (it is not acceptable to fail to follow through in security processes once established!)
+* Possibly distracts from other important work- we don't want to spread attention too wide too early
+* Adds bulk to Grin's governance structure
+* The community may trust core more than a team to make critical security decisions
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+* One alternative, currently in use, is adding the input of a security expert to the existing core decision making process
+  * This is better than nothing but we lose a few benefits without a dedicated security team:
+    * Strengthened security process: reduced likelihood of exploit leaks
+    * Improved transparency: there is currently little transparency around core's security decision making process
+    * Improved reaction times: informed, thoughtful, decisive actions with input from relevant experts only
+    * Possibility for security initiatives: clear decision making process for bilateral disclosure agreements, bug bounty programs, audit work
+    * Stronger commitments to the community, ecosystem and other projects: viable with a dedicated, clear decision making process
+    * Long-term scalability: extend capacity for security work to grow beyond the core team's attention capacity
+
+# Prior art
+[prior-art]: #prior-art
+
+grin-governance[0] and security-process[1] RFCs lay groundwork for the security-team RFC.
+
+# Unresolved questions
+[unresolved-questions]: #unresolved-questions
+
+- What are the security priorities of the Grin community and core team? (e.g. is privacy a higher priority than undetectable inflation?)
+- Is this the most minimal model that can have long-term success for Grin's security-related decision making process?
+- Is the single-veto model sufficient for a consistent, timely decision making process?
+- If core always has at least one appointed member, and one veto gives the decision to the core team, does this significantly weaken the security team's purpose?
```

Comment:
This might be redundant if there's not one always appointed member requirement

Reply:
Agreed and updated